docs/requirements-docs.txt

```diff
@@ -1,5 +1,5 @@
 numpydoc
-sphinx
+sphinx<3.5.0
```

Comment:
Per the discussion over in https://github.com/dask/dask/pull/7217

```suggestion
sphinx
```

Reply:
thanks, removed